solutions/deploy/cloud-pak-deployer/scripts/create-scc.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+${OC} adm policy add-scc-to-user "${SECURITY_CONTEXT_CONSTRAINT_NAME}" -z "${SERVICE_ACCOUNT_NAME}" -n "${NAMESPACE_NAME}"
```

Comment:
this can be achieved through a role binding resource directly as opposed to a bash scrip

Example:
```
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: 'system:openshift:scc:anyuid'
  namespace: <your-namespace>
subjects:
  - kind: ServiceAccount
    name: default
    namespace: <your-namespace>
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: 'system:openshift:scc:anyuid'
```

Reply:
this will also remove the need for the delete script